### DISCUSSION TITLE
Decrypting match file manually

### DISCUSSION THREAD

**Discussion by Brahma-Github** [fastlane/fastlane#21953] [Q&A]
I upgraded fastlane to latest version (2.220.0). I see file encryption and decryption logic updated from 'aes-256-cbc' to 'aes-256-gcm', and was suggested to make use of `match_file` for manual encryption / decryption.

How can I execute the companion script `match_file` to decrypt a file ?

**Comment by robert1997b** [comment]
Please note that the actual syntax and options for the match_file script may vary based on the implementation details of Fastlane’s update. If you encounter any issues or need further assistance with the specific syntax, I recommend checking the official Fastlane documentation or seeking support from the Fastlane community. [adp workforce now login](https://www.iadpworkforcenow.com) They can provide the most accurate guidance based on the latest version of the tool. Replace "path/to/encryptedfile" with the actual path to your encrypted file and "path/to/decryptedfile" with the desired output path for the decrypted file.
